CONTRIBUTING.md

```diff
@@ -173,3 +173,41 @@ Before merging a PR make sure:
 
 Merge the PR by using "Squash and merge" option on Github. Avoid creating merge commits.
 After the merge make sure referenced issues were closed.
+
+### Developing guidelines
+
+#### Building OCI Images for multiple arch (linux/arm64, linux/amd64)
+
+[OCI Images](https://github.com/opencontainers/image-spec/blob/master/spec.md) could be built and published by [buildx](https://github.com/docker/buildx), 
+it could be executed for local publish `all-in-one` images via:
+
+```shell
+TRAVIS_SECURE_ENV_VARS=true NAMESPACE=$(whoami) BRANCH=master ./scripts/travis/build-all-in-one-image.sh
```

Comment:
Why is this `TRAVIS_SECURE_ENV_VARS=true` needed? Isn't there a make target that could be used instead of the Travis script?

Reply:
I don't want to change too much in this pr.

We could refactor this in future